system/lib/compiler-rt/lib/lsan/lsan_common_emscripten.cc

```diff
@@ -78,14 +81,14 @@ void InitializePlatformSpecificModules() {
 }
 
 extern "C" {
-  extern int __data_end;
-  extern int __heap_base;
+  extern char __global_base;
+  extern char __data_end;
```

Comment:
What's the motivation for this name change?

Reply:
There is no name change. We no longer use `__heap_base` to identify the high end of the stack. The other change is that we scan from `__global_base` instead of `0`. On ASan, this will avoid scanning the large shadow memory.